examples/00_quick_start/sasrec-tf2.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.    #from recommenders.models.deeprec.io.nextitnet_iterator import NextItNetIterator
</pre><p>One question: <code>NextItNetIterator</code>  is an iterator that can be used in SASRec?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1530/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Not used here. Removed. </p>